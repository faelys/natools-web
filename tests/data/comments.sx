(elements
   (main-text (tags (comments
      "<ul>\n"
      (all-elements
         (limit 10)
         (render "<li>" (date (rfc-3339)) "</li>\n"))
      "</ul>\n")))
   (sidebar "")
   (title "Comment List"))
