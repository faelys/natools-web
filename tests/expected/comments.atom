<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
   <title>Test comments</title>
   <author>
      <name>Natacha Kerensikova</name>
   </author>
   <id>http://example.com/comment-feed</id>
   <updated>2015-02-15T08:09:10Z</updated>
   <entry>
      <id>http://example.com/first#VV118</id>
      <author>
         <name>Natasha Kerensikova</name>
      </author>
      <updated>2015-01-01T08:00:00Z</updated>
      <content type="html">
&lt;p&gt;Escaping to prevent HTML injection attacks:
&amp;lt;script&amp;gt;alert("hello");&amp;lt;/script&amp;gt;&lt;/p&gt;
      </content>
   </entry>
   <entry>
      <id>http://example.com/second#VV11AV</id>
      <author>
         <name>Natasha &lt;Faelys&gt; Kerensikova</name>
      </author>
      <updated>2015-01-01T10:31:00Z</updated>
      <content type="html">
&lt;p&gt;Happy new year everybody!&lt;/p&gt;
      </content>
   </entry>
   <entry>
      <id>http://example.com/first#VV17HO2</id>
      <author>
         <name>Anonymous</name>
      </author>
      <updated>2015-01-07T17:24:02Z</updated>
      <content type="html">
&lt;p&gt;Two comments on the same page, w00t \o/&lt;/p&gt;
&lt;p&gt;With even a second paragraph!&lt;/p&gt;
      </content>
   </entry>
   <entry>
      <id>http://example.com/first#VV2F89A</id>
      <author>
         <name>&lt;code&gt;Nat&lt;/code&gt;</name>
      </author>
      <updated>2015-02-15T08:09:10Z</updated>
      <content type="html">
&lt;p&gt;A third comment, with &lt;strong&gt;legitimate&lt;/strong&gt; embedded HTML&lt;/p&gt;
      </content>
   </entry>
</feed>
