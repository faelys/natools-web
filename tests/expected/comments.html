<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
  <head>
    <title>Comment List</title>
  </head>
  <body>
<h1>Comment List</h1>
<ul>
<li>2015-01-01T08:00:00Z on <a href="/first">First Page</a></li>
<li>2015-01-01T10:31:00Z on <a href="/second">Second Page</a></li>
<li>2015-01-07T17:24:02Z on <a href="/first">First Page</a></li>
</ul>
    <div class="sidebar">
    </div>
    <div class="footer">
<p>Default footer: <a href="/index">return to index</a></p>
    </div>
  </body>
</html>
